src/Parser/DeathKnight/Unholy/Modules/Items/ColdHeartEfficiency.js

```diff
@@ -20,63 +20,117 @@ class ColdHeartEfficiency extends Analyzer {
 
   totalColdHeartCasts = 0;
   correctColdHeartCasts = 0;
-  buffColdHeart = 0;
+  buffColdHeart = 20; //Stacks start at 20 at start of fight
+  
   unholyStrengthStart = 0;
   unholyStrengthRemaining = 0;
```

Comment:
My previous comment still stands after the changes, `unholyStrengthRemaining` `concordanceRemaining`  `khazgorothRemaining` and `timeAtMaxStacks` all should not be class variables because they are only set and immediately used within one function.